rskj-core/src/test/java/co/rsk/peg/lockingcap/LockingCapStorageProviderImplTest.java

```diff
@@ -0,0 +1,114 @@
+package co.rsk.peg.lockingcap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.peg.InMemoryStorage;
+import co.rsk.peg.lockingcap.constants.LockingCapConstants;
+import co.rsk.peg.lockingcap.constants.LockingCapMainNetConstants;
+import co.rsk.peg.storage.StorageAccessor;
+import java.util.Optional;
+import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+import org.ethereum.config.blockchain.upgrades.ActivationConfigsForTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class LockingCapStorageProviderImplTest {
+
+    private final LockingCapConstants constants = LockingCapMainNetConstants.getInstance();
+    private LockingCapStorageProvider lockingCapStorageProvider;
+    private ActivationConfig.ForBlock activations = ActivationConfigsForTest.all().forBlock(0);
+    private StorageAccessor bridgeStorageAccessor;
+
+    @BeforeEach
+    void setUp() {
+        bridgeStorageAccessor = new InMemoryStorage();
+        lockingCapStorageProvider = new LockingCapStorageProviderImpl(bridgeStorageAccessor);
+    }
+
+    @Test
+    void getLockingCap_whenNoPreviousValueExists_shouldReturnOptionalEmpty() {
+        // Arrange
+        Optional<Coin> expectedLockingCap = Optional.empty();
+
+        // Act
+        Optional<Coin> actualLockingCap = lockingCapStorageProvider.getLockingCap(activations);
+
+        // Assert
+        assertEquals(expectedLockingCap, actualLockingCap);
+    }
+
+    @Test
+    void getLockingCap_whenAPreviousLockingCapValueExistsInStorage_shouldReturnPreviousValue() {
+        // Arrange
+        Coin previousLockingCap = constants.getInitialValue().add(Coin.SATOSHI);
```

Comment:
previous to what?

Reply:
and why adding a SATOSHI to the `initialValue`? The initialValue is only being used in the LockingCapSupport. No related to these tests.

Reply:
I changed the name of the variable to `expectedLockingCap`.

Reply:
`About the second comment:` to avoid hard code an amount. I use the `initialValue` and add one Satoshi to express a value greater than the `initialValue`. It's easy to read and avoids magical numbers.